Adyen/Core/Components/Installment/InstallmentOptions.swift

```diff
@@ -85,41 +85,48 @@ public struct InstallmentConfiguration: Decodable {
     /// Options that are specific to given card types
     public let cardBasedOptions: [CardType: InstallmentOptions]?
     
-    /// Determines whether to show the money amount in the installment selection.
+    /// Determines whether to show the amount next to the installment value.
     /// For example, `3 months X 500 USD` or `3 months`.
-    /// For now it is disabled due to making the dividing calculations in the client.
+    /// Amount is calculated by simple division.
     @_spi(AdyenInternal)
-    public let showInstallmentPrice: Bool
+    public var showInstallmentAmount: Bool
     
     /// Creates a new installment configuration by providing both the card type based options
     ///  and default options for the rest of the card types.
     /// - Parameters:
     ///   - cardBasedOptions: Options based on the card type. Must not be empty.
     ///   - defaultOptions: Default options for cards that are not specified in `cardBasedOptions`.
-    public init(cardBasedOptions: [CardType: InstallmentOptions], defaultOptions: InstallmentOptions) {
+    ///   - showInstallmentAmount: Determines whether to show the amount next to the installment value.
+    public init(cardBasedOptions: [CardType: InstallmentOptions], 
+                defaultOptions: InstallmentOptions,
+                showInstallmentAmount: Bool = false) {
         assert(!cardBasedOptions.isEmpty, "This dictionary must not be empty.")
```

Comment:
Maybe a good one to think about (in a different PR) either making the init throwing or optional to expose a possible issue.